### ISSUE TITLE
Take adjusted argument out of read.R

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#146]
As discussed in our vftf today, the adjusted argument no longer needs to be in `read.R()`.

**Comment by dankelley** [comment]
@j-harbin can you help me with this?  In "develop" commit db55a565a4fe1891cc01bc65fda3cb2fc9aace93, I have new code that tries to address this issue, but it is not correct yet (and it spits out diagnostic values).

I want to do a test, of course, and want to use a netcdf file that we supply with the package (as always, we do not want tests to download data).  

However, I think that neither of the two netcdf files in our package is suitable  To see this, you could do
```R
library(argoFloats)
A <- system.file("extdata", "D4900785_048.nc", package="argoFloats")
B <- system.file("extdata", "SR2902204_131.nc", package="argoFloats")
aA <- readProfiles(A)
aB <- readProfiles(B)
```
to read in both of our built-in files, then do
```R
summary(aA[[1]])
summary(aB[[1]])
```
to summarize those profiles.  In the first case, I am seeing that the "Adjusted" variables match the unadjusted ones.  And in the second, the "Adjusted" ones are all NA. (Maybe I'm wrong?)

Therefore, I'd like to find a file for which adjustment will make a useful change.  And I hope you can help.

Can you try to find such a file, and post here the code to get it?  I prefer it to be listed in one of our built-in 'data' index files, because then I can write reproduceable code in the `create_data` directory to download the data file. It doesn't matter much what the file is about -- we just need something with non-`NA` "adjusted" values, an with the unadjusted and adjusted values differing.

**Comment by j-harbin** [comment]
Dan, I recommend you use the file we sent to Tanya. You can obtain this file from the following code and you'll see that `<PARAM>_ADJUSTED>` is in fact adjusted. 

```R
library(argoFloats)
bai <- getIndex('synthetic')
s <- subset(bai, ID='5903586')
ss <- subset(s, 50)
profiles <- getProfiles(ss)
argos <- readProfiles(profiles)
a <- argos[[1]]
```

**Comment by dankelley** [comment]
Thanks.  I was having a very hard time remembering where we found such a file!  You saved me a lot of fiddling around!!

**Comment by dankelley** [comment]
done
